PREREQUISITES.md

```diff
@@ -370,8 +370,8 @@ problems.
   mitigation applied on SGX CVEs as documented here [sgxsdk-guide](#https://github.com/intel/linux-sgx)
 
   ```bash
-  wget https://download.01.org/intel-sgx/latest/linux-latest/as.ld.objdump.gold.r2.tar.gz
-  tar -xvf as.ld.objdump.gold.r2.tar.gz
+  wget https://download.01.org/intel-sgx/latest/linux-latest/as.ld.objdump.gold.r3.tar.gz
```

Comment:
Can we use the files relevant to the SGX release we use  for AValon. I assume we support 2.10. If so, please use files in https://download.01.org/intel-sgx/sgx-linux/2.10/
eg: https://download.01.org/intel-sgx/sgx-linux/2.10/as.ld.objdump.gold.r2.tar.gz

Reply:
Another option is to use latest 2.12. In that case use files from

https://download.01.org/intel-sgx/sgx-linux/2.12/

Reply:
Earlier there was no sgx version specific mitigation tools available, hence we used latest. Now I do see, mitigation tools in https://download.01.org/intel-sgx/sgx-linux/2.10/ link. will try to use the same.